epix/event_separation.py

```diff
@@ -30,14 +30,16 @@ def times_from_fixed_rate(rate, n_events, offset):
 
     Args:
         rate (int or float): Mean event rate in Hz. 
-        n_events (int): Number of events
+        n_events (int): Number of events in file
+        n_events_simulated (int): True number of events which were
+            simulated.
```

Comment:
You are making this distinction to separate between the event rate that can be found in the detector and the actual rate at the (external) source or?

Reply:
n_events is needed to know how many numbers we need, n_events_simulated is required to get the correct source rate in the end.